dockerfiles/Dockerfile.openvino

```diff
@@ -30,12 +30,13 @@ ENV LANG en_US.UTF-8
 
 RUN apt update && \
     apt -y install git sudo wget locales \
-    zip x11-apps lsb-core cpio libboost-python-dev libpng-dev zlib1g-dev libnuma1 ocl-icd-libopencl1 clinfo libboost-filesystem1.58.0 libboost-thread1.58.0 protobuf-compiler libprotoc-dev libusb-1.0-0-dev autoconf automake libtool && \
+    zip x11-apps lsb-core cpio libboost-python-dev libpng-dev zlib1g-dev libnuma1 ocl-icd-libopencl1 clinfo libboost-filesystem1.58.0 libboost-thread1.58.0 protobuf-compiler libprotoc-dev libusb-1.0-0-dev autoconf automake libtool libudev-dev libjson-c-dev && \
     cd ${MY_ROOT} && \
     git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO onnxruntime && \
     cp onnxruntime/docs/Privacy.md /code/Privacy.md && \
     cp onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt && \
     cp onnxruntime/ThirdPartyNotices.txt /code/ThirdPartyNotices.txt && \
+    sed -i "s/sudo//" onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
```

Comment:
why is this needed?
is it just to get rid of the sudo at https://github.com/microsoft/onnxruntime/blob/9f7d2454463e19f585f3acd75e2ca5b128a88a55/dockerfiles/scripts/install_common_deps.sh#L22 ?
if so, can we modify that script and remove the sudo when downloading cmake? i'm not sure why it's needed anyway.

Reply:
This is mainly needed to be able to validate container images behind company proxy servers and with conda python3 envs. There is a 'sudo pip install numpy' within the install_common_deps.sh, which we would like to use pip3 from the conda env. However, using sudo would cause it to use ubuntu 16.04's python2.7's pip instead. An alternative could be to replace sudo with 'sudo -E'.

Reply:
Where do you see sudo pip install numpy in install_common_deps.sh ? There's no sudo in the latest version on master.

Reply:
Ah.. my mistake.. i guess folks were having issues with line 66 (wget) behind proxy.

Reply:
yes, then we should get rid of the sudo in line 22. not sure why it's even there.
there isn't one for the wget in line 14.

Reply:
Sounds good. I have removed the sed replacement line to remove sudo from install_common_deps.sh

Reply:
Will you make that change to remove the sudo from the wget in install_common_deps.sh ?
otherwise you mentioned you were running into issues?